<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="QT-INNO-LT0204" failures="1" tests="3" name="Default test" time="11.854" errors="0" timestamp="17 Apr 2021 15:34:42 GMT">
  <testcase classname="com.classplus.assignment.price_comparison.suites.PhonePriceTest" name="TC01_fetchPhonePriceAmazon" time="4.551">
    <failure type="java.lang.AssertionError" message="[TEST FAILED]: Product &amp;apos;iPhone XR 64GB  Yellow&amp;apos; is not displayed in Amazon 
 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: [TEST FAILED]: Product 'iPhone XR 64GB  Yellow' is not displayed in Amazon 
 expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.classplus.assignment.price_comparison.pages.PhoneComparisonKeyword.fetchPhonePriceAmazon(PhoneComparisonKeyword.java:63)
	at com.classplus.assignment.price_comparison.suites.PhonePriceTest.TC01_fetchPhonePriceAmazon(PhonePriceTest.java:19)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:882)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1189)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC01_fetchPhonePriceAmazon -->
  <testcase classname="com.classplus.assignment.price_comparison.suites.PhonePriceTest" name="TC02_fetchPhonePriceFlipkart" time="3.459"/>
  <testcase classname="com.classplus.assignment.price_comparison.suites.PhonePriceTest" name="TC03_comparePrices" time="0.001"/>
</testsuite> <!-- Default test -->
